### DISCUSSION TITLE
读《以利为利》 | /posts/zfz/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#599] [Announcements]
https://yuanfan.rbind.io/posts/zfz/

**Comment by botsp** [comment]
R1： "有可能使得政府成为一个与人民利益相对独立的利益集合体。"
> - 有些时候互相独立、而另一些时候甚至对立了。

R3: 考虑到这本书是2012年出版的，确实珍贵、原来关于问题与答案的讨论已经是那样明确了，但又感觉似乎现在的环境已经无心去关注那时的研究了。人们尚未认真观察、思考那个刚刚走远的昨日，就被剧烈的卷入一个更加疯狂的时代了。现在大家有更烦恼的事情，单单就这三年的纷纷扰扰就值得未来的无数本专著。

看到一个讲法，如果从"民主"的角度解读那三十年、很难解释为什么没有选票还有那样辉煌的成绩。但是换个说法，从"权力的分散"来看就一切讲得通了: 经济权让渡给市场、司法权让渡给法律、党权让渡给行政、定于一尊让渡给多人共治，从这个角度既能解释由坏变好、也能理解为什么由好变坏了。

**Reply by earfanfan** [reply]
R1和 R3的 R 是指什么？R2去哪了呢。

>原来关于问题与答案的讨论已经是那样明确了，但又感觉似乎现在的环境已经无心去关注那时的研究了。人们尚未认真观察、思考那个刚刚走远的昨日，就被剧烈的卷入一个更加疯狂的时代了。

这么想也对，不过大家可能都挺“身边即世界”的。按照我的理解，学界、政界的联系相对紧密多了，但是这两部分人都属于如果知晓实际历史了的话就没法明确说出来的，现在我才知道的很多问题其实在学界已经争论过许多年了，只是因为我没有身在其中，所以不知道，其实还是有很多人在研究历史关注现在也展望未来的，也有很多人在继续认真观察和思考。人类历史上一定经历过许多超级重大的天灾人祸，有人在作恶，有人顾自己，也有人会奉献，世界上的人有很多不同的组成。其实，每个时代都挺疯狂的，只不过咱们大家都受到自己身处时代的局限，所以感受不到以前多疯狂，也看不见未来多疯狂。尽管现在网络上随便点开哪都能看到大家吵翻了天，不过真的不用对人类失去信心，哈哈，毕竟能够拯救全人类的还得是人类自己。嗷，对了，我默认你跟我一样，身处民间，不属于学界也不属于政界。

很多现在的问题绕不开一些历史原因，所以有不少经济学、社会学的经典书籍现在也还很值得看，我是先看了2022年出版的《置身事内》，再看的2012年出版的《以利为利》，但是感觉前者的内容有不少都是参考的后者。所以呀，那时的研究价值现在看来也并未褪色多少呢。

你看到的这个观点挺有道理的。

**Reply by botsp** [reply]
看完长文还整理了一二三的读后感，后来想了想还是慎言、就删掉了第二点，哈哈。

第二段严重同意，怎样构建自己的理解、想象，怎样定义客观的真实，值得智者的思考。

在现在这样的环境下很容易陷入宏大的焦虑中，能平静得看一些讨论具体问题的书真的难得。总觉得清王朝亡之前、讨论戊戌变法的方法学，试图从方法、技术上改良执政策略，是对大清人民的残忍，倒不如一切推到重来。来自一位悲观的激进主义者的碎碎念。

**Reply by earfanfan** [reply]
哈哈，世界是什么样子的，并不只是值得智者的思考，你这样说得好像我是个智者似的，其实像我这样的普通人也有超多笨蛋时刻。互联网时代使得每个人每天都接触到大量信息，每个人都在思考和构建自己想象中的真实，我昨天还听我小学同学讲“认知是认知、事实是事实”。

>总觉得清王朝亡之前、讨论戊戌变法的方法学，试图从方法、技术上改良执政策略，是对大清人民的残忍，倒不如一切推到重来。来自一位悲观的激进主义者的碎碎念。

唔，你的历史观好像跟我的一个朋友有点相似，他叫湘云，他的博客地址是<https://xiangyun.rbind.io/daily/>，有时间的话你不妨去他那里溜达溜达，说不定你俩也能谈得来。

本怂人此次愿意鼓起勇气斗胆用一点自身经历来吐槽一下“推到重来”，其实我是最近几年开始接触一些女性主义的思想的，最初的最初先接触到的就是激进女权，可能也是重症就该下猛药不然不生效的缘故，激进女权那种有些类似一切推倒重来的想法让我一下子从蒙昧无知中开始走上觉醒的道路，但这条路也接近于是要不断地用刀把身上腐烂的血肉全都剜下来然后再重新生长，哈哈，真得超级超级疼的说。对于我这一个人来说，我可以去走这样的路，被激进女权唤醒后，一步步重塑，然后再重新选择希望全世界不论任何性别一起联合起来追求自由平等的思想，当然我的觉醒也还没结束，仍然还在继续寻求思想解放的路上摸索着。但是对于一个国家来说，让一个国家的女性全都同时一口气这么干，真得会很疯狂，因为人群中存在极大的参差，再加上不同年龄段也有一些固守观念难以改变，这样不知道会一下子剥夺多少人的幸福。全部人思想上的改变恐怕真得不得不是渐进式的，需要时间来慢慢修正的。

再回到清朝的变法和改革，当时做不到一切推到重来，正如现在也无法做到一切推倒重来一样。但是呢，其实《以利为利》里面有讲述分税制改革之前的历史，当时的财政包干制让中央的财税收入降到了一个地步、地方的独大也到了一个地步，已经到了必须改革的地步，于是中央政府下决心推动了分税制改革。清朝的变法和改革，主要发动者也是整个大清帝国的最高权力，所以他们选择的也是改革，是修修补补而不是推到重来。

你说现在的环境容易陷入宏大焦虑，我感觉这件坏事里面也有一点好，可能踩到坑里再靠自己努力爬出来后，大家才会明白如何克制受到环境的影响而减少焦虑，就当做时代环境也是生活在这个时代的人都必须面对的一项大考验吧。另外，我感觉你的心态也蛮好的，不过好像也提过几次环境的影响，希望时代给你的考验也是在催化你的进步，而不是只是让你陷入深渊吧。

**Reply by botsp** [reply]
女权激进也不是不行的，哈哈。女性被压迫几千年了，翻个身压迫男性几百年也未尝不可嘛。先追求激进、至少是口号激进，再退而求温和，也是一种积极的策略。俺经常在女性领导手下干活儿，只被她们的无限魅力折服，对此毫无意见，emmmmm。

对于群体层面来说，很赞同你的看法。追求女权之前、不如先号召追求人权。不然某些呼喊，对于性别歧视的现象奔走哭嚎，但是对于某些更为残酷的现象似乎无动于衷置身事外(俺的个人感受)，观感实在不雅。即使从现实角度看，没有人权的保障，女权面临的困难就会更大些。

感谢推荐、稍微瞄了一眼很有同志之慨，马上follow！

**Reply by earfanfan** [reply]
那个，其实在我现在的观念里，也不认同让女性压迫男性……职场跟生活还是不一样的，因为职场本来就是要去除每个个体的独特个性全都变成一样的螺丝钉，职场里本来就充斥着各种等级森严的规则，并且不论性别，都是强权者对弱权者的压迫。

你在上一楼写的第二段让我又想给你再安利几个博主了，因为正是在我观念偏激进的时候，是他们告诉我还应该关注女权之外的领域。不过我这两天给你安利的已经太多了，还是等你自己慢慢发现他们吧。

**Reply by botsp** [reply]
哈哈，感谢，到处逛一逛。感觉现在反而倒退回了古典互联网时代，博客、bbs这些，信息质量更高、氛围也更好，咱们携手"返古"吧！

**Reply by earfanfan** [reply]
你说的这个古典互联网时代我好像没撒印象，相反，对我而言独立博客相对微博、朋友圈这类更加新。嘿，哈哈哈，你是不是暴露年龄啦，我感觉你可能是80后。现在应该是一个信息载体丰富并存的时代，长文字、短文字、长视频、短视频都有受众的。
